test/OpenTelemetry.Instrumentation.AspNetCore.Tests/BasicTests.cs

```diff
@@ -496,6 +496,60 @@ void ConfigureTestServices(IServiceCollection services)
             Assert.Equal(shouldEnrichBeCalled, enrichCalled);
         }
 
+        [Fact]
```

Comment:
Consider making this parameterized:
1. suppress instrumentation = true, nothing gets called
2. suppress instrumentation = false, processor/propagator are called